sdk/search/Microsoft.Azure.Search/CHANGELOG.md

```diff
@@ -42,7 +42,7 @@ None
 
 ### Breaking Changes
 
-For more details, visit the [SDK migration guide](https://docs.microsoft.com/azure/search/search-dotnet-sdk-migration-version-10).
+For more details, visit the SDK migration guide.
```

Comment:
Did we intend to remove the link to the migration guide?   Seems that'd be helpful!

Reply:
These links are broken and don't exist.

Reply:
Is this the one you want - https://docs.microsoft.com/en-us/azure/search/search-dotnet-sdk-migration-version-11?